build/migration/bin/2_previous-integration-test.sh

```diff
@@ -34,10 +34,34 @@ run_tests(){
         echo "Done Running Tests"
         echo ""
 
-        # docker container logs "db2_fhir_1" 
+        # docker container logs "db2_fhir_1"
     fi
 }
 
+clean_up(){
+    # Remove problematic resources
+    fhir_base_url=https://localhost:9443/fhir-server/api/v4
+    problem_types=CompartmentDefinition
+    for r in $(curl -k -u fhiruser:change-password "${fhir_base_url}?_type=${problem_types}&_count=1000" | jq -r '.entry[].fullUrl'); do
+      echo "erasing ${r}"
+      curl -i -k -u fhiruser:change-password -H "Content-Type: application/json" ${r}'/$erase' -d '
+        {
+          "resourceType": "Parameters",
+          "parameter": [
+            {
+              "name": "patient",
```

Comment:
what relationship does patient `test` have with CompartmentDefinition? Could use a comment to indicate how this value relates to the code.

Reply:
How are errors handled?

Reply:
> what relationship does patient test have with CompartmentDefinition? Could use a comment to indicate how this value relates to the code.

none.  i thought this parameter was required but apparently its only required for resource types that can be in the patient compartment.  i could remove it, but if we ever want to delete resource types that *are* in the patient compartment we'd need to add it back in

Reply:
> How are errors handled?

they're not.  it would pass this step and move on to our e2e tests and those might fail.  that would be confusing, but not catastrophic.
i think i could update the curl command to return an error code if the request was unsuccessful...that would lead to a much more clear error in the ci job logs

Reply:
ok, i pushed an update that includes the `--fail` flag.  combined with the `set -ex` at the beginning of the script, that should ensure we exit if one of these requests fail (which they shouldn't)